src/frequenz/dispatch/__init__.py

```diff
@@ -9,7 +9,11 @@
 
 from frequenz.dispatch.actor import DispatchActor, DispatchEvent
 
-__all__ = ["Dispatcher"]
+DispatchReceiver = Receiver[Dispatch]
+"""Type alias for a dispatch receiver."""
+
+DispatchEventReceiver = Receiver[DispatchEvent]
+"""Type alias for a dispatch event receiver."""
```

Comment:
Is this really related to this commit? If so maybe mention it in the commit message?

Reply:
"Make types available"?

Reply:
I have rephrased it for your liking